README.md

```diff
@@ -57,6 +66,10 @@ reverse(evens)
 console.log(evens) // [10, 4, 6, 8, 2]
 
 function reverse(array) {
-  // -- YOUR ANSWER HERE --
+  let j = array.length;
+  for(let i = 0; i < array.length; i++){
+    swap(array, i, j)
+    j -= 1;
+  }
```

Comment:
Nice.  But do you get the results that you expect?  Remember that at `array.length/2`, `i` and `j` meet in the middle....